model/src/main/proto/exploration_checkpoint.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
```

Comment:
This file needs to be declared here as well 
https://github.com/oppia/oppia-android/blob/develop/model/BUILD.bazel
@BenHenning correct?

Reply:
done